hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -48,12 +48,12 @@ public record TokensConfig(
                 ScaleFactor nftsMintThrottleScaleFactor,
         @ConfigProperty(value = "nfts.useVirtualMerkle", defaultValue = "true") @NetworkProperty
                 boolean nftsUseVirtualMerkle,
-        @ConfigProperty(value = "maxPendingAllowedAirdrops", defaultValue = "20000000") @NetworkProperty
-                long maxPendingAllowedAirdrops,
-        @ConfigProperty(value = "maxAllowedPendingAirdropsToCancel", defaultValue = "10") @NetworkProperty
-                long maxAllowedPendingAirdropsToCancel,
+        @ConfigProperty(value = "maxAllowedAirdrops", defaultValue = "20000000") @NetworkProperty
```

Comment:
We should probably include `Pending` in the name. I think Atul suggested the name should have pending

Reply:
Agreed, fixed.